src/main/resources/config/liquibase/master.xml

```diff
@@ -80,4 +80,6 @@
     <include file="classpath:config/liquibase/changelog/20200728142700_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20200805101210_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20200806141810_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20200908112330_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20200921155030_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
This can be one migration!

Reply:
8111f8f1b9965ef189e6be645b839d9338416cf0